### DISCUSSION TITLE
Welcome to Nautobot Discussions!

### DISCUSSION THREAD

**Discussion by lampwins** [nautobot/nautobot#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by routenull0** [comment]
Is it possible to get rid of the ASCII art on this?
 
NTC (╯°□°)╯︵ ┻━┻ 

It is very unpleasant to look at.

**Reply by glennmatthews** [reply]
It was a bit of silliness in celebration of all the work we'd put in as a team to get to 1.0, but on reflection we realize it might have given the wrong impression. As part of the work in restoring the NetBox commit history to this repository, we've amended that commit message as well.
